pkg/pods/pod_test.go

```diff
@@ -29,7 +28,9 @@ import (
 )
 
 func getTestPod() *Pod {
-	return NewPod(types.PodID("hello"), "testNode", "/data/pods/test")
+	// Can't use public method because pod home is different than pod id
```

Comment:
what is the public method that you can't use? I only see exported methods being used here, so I think I don't understand the comment